test/unit/ngsiv2/MQTT_receive_measures-test2.js

```diff
@@ -403,4 +403,32 @@ describe('MQTT: Measure reception ', function () {
             });
         });
     });
+    describe('When a POST single Raw measure arrives for the HTTP binding', function () {
+        beforeEach(function () {
+            contextBrokerMock
+                .matchHeader('fiware-service', 'smartgondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post(
+                    '/v2/entities?options=upsert',
+                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/singleMeasuresRawTypes1.json')
+                )
+                .reply(204);
+        });
+        it('should send its value to the Context Broker', function (done) {
+            mqttClient.publish('/1234/MQTT_2/attrs/humidity', '0x4D', null, function (error) {
+                setTimeout(function () {
+                    contextBrokerMock.done();
+                    done();
+                }, 100);
+            });
+        });
+        it('should return a 200 OK with no error', function (done) {
+            mqttClient.publish('/1234/MQTT_2/attrs/humidity', '0x4D', null, function (error) {
+                setTimeout(function () {
+                    contextBrokerMock.done();
+                    done();
+                }, 100);
+            });
+        });
```

Comment:
This tests does not have sense (MQTT broker does not return any HTTP code)

Reply:
Hi @mapedraza , should I remove this.

Reply:
Hi @mapedraza , I have removed these lines.